src/main/java/picard/fingerprint/HaplotypeProbabilityOfNormalGivenTumor.java

```diff
@@ -51,12 +58,13 @@ public TransitionMatrix(double pLoH) {
                     {0, 0, 1}}; //normal is hom_var => tumor must be the same
         }
 
-        public double[][] getTransitionMatrix() {
+        double[][] getTransitionMatrix() {
             return transitionMatrix;
         }
     }
 
     private final HaplotypeProbabilities hpOfTumor;
+    @SuppressWarnings("MismatchedQueryAndUpdateOfCollection")
     static private Map<Double, TransitionMatrix> transitionMatrixMap = new CollectionUtil.DefaultingMap<>(TransitionMatrix::new, true);
```

Comment:
This seems dangerous to me. For one, it's not thread safe. It also introduces state which is never cleared, e.g. a memory leak, if the code is ever used in a non-CLP situation.